onmt/modules/GRU.lua

```diff
@@ -28,7 +28,7 @@ Parameters:
   * `layers` - Number of layers
   * `inputSize` - Size of input layer
   * `hiddenSize` - Size of the hidden layers
-  * `dropout` - Dropout rate to use (should be in $$[0,1]$$ range.
+  * `dropout` - Dropout rate to use (in $$[0,1]$$ range, if negative, force dropout even on first layer).
```

Comment:
not sure we want to do that since it won't be used by end-user. We are just stacking Encoders in complex encoders - and only in these cases, we will want to have first layers adding a dropout